rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/dataspace/dto/ListFile.java

```diff
@@ -36,15 +36,16 @@
  */
 package org.ow2.proactive_grid_cloud_portal.dataspace.dto;
 
-import java.util.List;
-
 import org.codehaus.jackson.map.annotate.JsonSerialize;
 
+import java.util.List;
+
 
 @JsonSerialize(include = JsonSerialize.Inclusion.NON_EMPTY)
 public class ListFile {
     private List<String> directories;
     private List<String> files;
+    private List<String> all;
```

Comment:
I could rename "All" to "DirectoriesAndFiles". But I cannot simply add as you suggest the content of directories and files. This is because here we don't talk about Sets but Lists. The order is important, in my "all" list I can have : [dir1,file1,dir2,file2] and I want to keep the order as it is.